application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -144,56 +140,51 @@ public interface TransitService {
    * stop, area stop or stop group, then a list with one item is returned.
    * An empty list is if nothing is found.
    */
-  Collection<StopLocation> getStopOrChildStops(FeedScopedId id);
+  Collection<StopLocation> findStopOrChildStops(FeedScopedId id);
```

Comment:
My intuition here would be to use get because the argument is an id.

Reply:
Collection is `find`